src/Handler/CurlFactory.php

```diff
@@ -63,7 +63,7 @@ public function release(EasyHandle $easy)
         $resource = $easy->handle;
         unset($easy->handle);
 
-        if (count($this->handles) >= $this->maxHandles) {
+        if (is_array($this-handles) && count($this->handles) >= $this->maxHandles) {
```

Comment:
Missing > in this - handeles

Reply:
the count function does support null in php 7.2 .

Reply:
This works for js_composer

Reply:
is_countable($this->handles)